content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,460 @@ linkTitle: "Workload Manager"
 type: docs
 weight: 70
 description: >
-  Workload Manager 
+  Workload Manager
 ---
 
-{{% pageinfo %}}
-This is a placeholder page.
-{{% /pageinfo %}}
+# Table of Contents
+
+* [What is the EGI Workload Manager?](#what-is-the-egi-workload-manager)
+* [Main Features](#main-features)
+* [Target User Group](#target-user-groups)
+* [Architecture](#architecture)
+* [How to access the EGI Workload Manager service](#how-to-access-the-egi-workload-manager-service)
+* [Getting Started](#getting-started)
+  * [Submit a service order via the Marketplace](#submit-a-service-order-via-the-marketplace)
+  * [Before to start](#before-to-start)
+    * [Apply for your user credentials](#apply-for-your-user-credentials)
+    * [Export your credentials from your browser](#eport-your-credentials-from-your-browser)
+    * [Install your credentials](#install-your-credentials)
+    * [Send your certificate's subject to the DIRAC team](#send-your-certificates-subject-to-the-dirac-team)
+  * [The EGI Workload Manager Web Portal](#the-egi-workload-manager-web-portal)
+    * [Upload Proxy](#upload-proxy)
+    * [Job Submission](#job-submission)
+    * [Monitor Job status](#monitor-job-status)
+    * [Get Results from Sandbox](#get-results-from-sandbox)
+    * [Full User Guide for DIRAC Web Portal](#full-user-guide-for-dirac-web-portal)
+  * [The DIRAC client tool](#the-dira-client-tool)
+    * [Managing simple jobs](#managing-simple-jobs)
+    * [Jobs with Input Sandbox and Output Sandbox](#jobs-with-input-sandbox-and-output-sandbox)
+    * [More details](#more-details)
+    * [Technical Support](#technical-support)
```

Comment:
I'm a bit surprised to see a manual table of content here, isn't the one provided by the theme enough?

Reply:
See the page ToC in the right column
 [](url)
![Screenshot 2020-10-12 at 15 37 40](https://user-images.githubusercontent.com/451417/95752821-ec148300-0ca0-11eb-9400-df2bcc4a20b9.png)

Reply:
I didn't notice the index.
We are perfectly fine with it.

Reply:
55f68673649d0c4aac8a7ccdb3ccd3a337405989

Reply:
Somehow the TOC is still there, not sure on how that's possible